sdk/keyvault/keyvault-common/package.json

```diff
@@ -57,7 +57,9 @@
   },
   "dependencies": {
     "@azure/core-http": "^1.2.0",
-    "tslib": "^2.0.0"
+    "tslib": "^2.0.0",
+    "@azure/core-tracing": "1.0.0-preview.9",
+    "@opentelemetry/api": "^0.10.2"
```

Comment:
Rush was complaining because I didn't have this.

Reply:
keyvault-common gets required directly by the other keyvault clients. These dependencies don't alter the other keyvault clients artifacts

Reply:
Why did it not complain until now?